bin/inx-chronicle/src/api/mod.rs

```diff
@@ -131,6 +131,10 @@ impl Actor for ApiWorker {
         log::info!("Stopping API server");
         run_result
     }
+
+    fn name(&self) -> std::borrow::Cow<'static, str> {
```

Comment:
question: the name stays the same whole lifetime of the app ('static), right, so why make this a Cow rather than just return a `&'static str`?

Reply:
Forget it....I see that it sometimes needs to own a dynamically created value.